src/services/member/plugins/action/test/service.test.ts

```diff
@@ -37,19 +38,22 @@ describe('Action member service', () => {
   });
 
   describe('Test getFilteredActions', () => {
-    it('throw for signed out user', async () => {
-      ({ app } = await build({ member: null }));
+    // FIX this test is now obsolete, as the endpoint checks the member authentication
+    // the type should remove the need to check
+    // it('throw for signed out user', async () => {
+    //   ({ app } = await build({ member: null }));
 
-      await expect(
-        getActionMemberService().getFilteredActions(actor, buildRepositories(), {}),
-      ).rejects.toBeInstanceOf(UnauthorizedMember);
-    });
+    //   await expect(
+    //     getActionMemberService().getFilteredActions(actor, buildRepositories(), {}),
+    //   ).rejects.toBeInstanceOf(UnauthorizedMember);
+    // });
```

Comment:
I think we can delete this test, right ? No needs to keep it

Reply:
Yes will delete it !